### DISCUSSION TITLE
blog/adding-videos-to-keystatic-content-with-astro/

### DISCUSSION THREAD

**Discussion by giscus** [armno/armno.in.th#53] [Blog Post Comments]
# blog/adding-videos-to-keystatic-content-with-astro/

My notes on building a custom component in Keystatic using Content Components feature for my Astro blog

https://armno.in.th/blog/adding-videos-to-keystatic-content-with-astro/

**Comment by cisternascristian** [comment]
I want you to know how helpful this blog was for me, it took me hours to crack how to implement something simplier than this, because the Keystatic docs aren't that good in this matter especially because they deprecated the document field. Anyways, you're a life saver, all your process was all i needed.

# Thank you so much! Keep up the good work!

**Comment by thewhiteh4t** [comment]
Thanks for this! helped a lot for my use case
